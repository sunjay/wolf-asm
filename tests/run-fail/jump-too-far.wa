# This programs jumps past the end of memory, resulting in an out of bounds
# exception in the VM

section .code

jmp 64($sp)
