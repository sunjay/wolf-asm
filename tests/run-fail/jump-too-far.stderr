Error: Failed to execute instruction at `0x1038`

Caused by:
    Invalid memory access: attempt to access `0x1038` when address must be less than `0x1000`
